synapse/server_notices/resource_limits_server_notices.py

```diff
@@ -91,24 +92,28 @@ def maybe_send_server_notice_to_user(self, user_id):
         currently_blocked, ref_events = yield self._is_room_currently_blocked(room_id)
 
         try:
-            if not self._config.mau_limit_alerting:
-                # Alerting disabled, reset room if necessary and return
+            is_auth_blocking = False
```

Comment:
this feels pretty redundant - indeed, you don't bother to check it at line 110. Let's just get rid of it and use `event_limit_type` instead.